api/src/main/resources/firecloud.yaml

```diff
@@ -1854,6 +1893,12 @@ components:
         allUsersGroup:
           type: boolean
           description: User is a member of the "All Users" group?
+        adminEnabled:
+          type: boolean
+          description: User is a member of the admin group?
+        tosAccepted:
+          type: boolean
+          description: User has accepted the latest Terms of Service?
```

Comment:
Consider using "latest" or "current," but not both if they are intended to mean the same thing.

Reply:
this is copied from Orchestration.

Are you saying that I should use "latest" throughout when referring the Terra TOS version we need to match?

Reply:
(I did this)